### PR TITLE
add contributing guidelines

### PR DESCRIPTION
**Pull request by tkellen**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Issue Tracker
+> http://github.com/json-api/json-api/issues
```
Comment (steveklabnik) [inline CONTRIBUTING.md:2]: does this actually link in markdown output? also, there probably should be a newline between this and the first line. same on line 13

**Comment by steveklabnik** [comment]
:+1: with the small nit

**Comment by dgeb** [comment]
:+1: Let's wait to merge until we get the initial roadmap on the about page.

**Comment by tkellen** [comment]
@steveklabnik https://github.com/tkellen/test/blob/master/CONTRIBUTING.md

**Comment by tkellen** [comment]
Added the roadmap.

File: CONTRIBUTING.md (hunk 1 above)
Comment (tkellen) [inline CONTRIBUTING.md:2]: Yup, it does, see the test repo I linked.

**Comment by ethanresnick** [comment]
:+1:

**Pull request merged**